### PR TITLE
Multiple issues addressed

### PR DESCRIPTION
**Pull request by markkramerus**
See individual commit messages for the fixes in this PR.

### CODE REVIEW

File: input/fsh/AL_CodeSystems.fsh [hunk 1]
```diff
@@ -4,6 +4,7 @@ Alias:   SCT = http://snomed.info/sct
 Alias:   UCUM = http://unitsofmeasure.org
 Alias:   NCIT = http://ncithesaurus-stage.nci.nih.gov
 Alias:   ICD10CM = http://hl7.org/fhir/sid/icd-10-cm
+Alias:   ICD10PCS = http://www.cms.gov/Medicare/Coding/ICD10
```
Comment (mlterryMitre) [inline input/fsh/AL_CodeSystems.fsh:7]: probably not correct URL although there is no consensus at this time.  https://chat.fhir.org/#narrow/stream/179175-argonaut/topic/ICD-9.20and.20ICD-10.20system.20URIs/near/192479214.
I suppose the closest one for now until there is resolution is to align with what US Core put in for R4. 
http://www.icd10data.com/icd10pcs

**Review by masnick — APPROVED** [review]

**Review by mlterryMitre — COMMENTED** [review]
Maybe it's because I'm finally getting to reviewing this in detail, but there are a number of modeling changes that is breaking backwards compatibility and additions of new profiles that have flown under the radar.
I'll approve it just not to be a blocker, but it might make sense to revisit an overall conceptual diagram of what exactly we want in STU2 rather than to put a lot of effort into making these design changes and figure what it means for overall changes after the fact.

File: input/fsh/EX_Example1.fsh [hunk 2]
```diff
@@ -9,7 +9,7 @@ Description: "mCODE Example for Primary Cancer Condition"
 * code = SCT#254637007 "Non-small cell lung cancer (disorder)"
 * extension[histologyMorphologyBehavior].valueCodeableConcept = SCT#35917007 "Adenocarcinoma"
 * bodySite = SCT#39607008 "Lung structure (body structure)"
-* bodySite.extension[laterality].valueCodeableConcept = SCT#7771000 "Left (qualifier value)"
+* bodySite.extension[locationQualifier].valueCodeableConcept = SCT#7771000 "Left (qualifier value)"
```
Comment (mlterryMitre) [inline input/fsh/EX_Example1.fsh:12]: This is a breaking change. Is there a clear rationale why we want to do this other than a name preference?

**Review by mlterryMitre — APPROVED** [review]

File: input/fsh/EX_Example1.fsh (hunk 2 above)
Comment (mlterryMitre) [reply input/fsh/EX_Example1.fsh:12]: approved with noted comments that probably need revisiting in subsequent updates.

**Pull request merged**